src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.*;
+import core.basesyntax.enums.Type;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX = 20;
+    private static final int MIN = 1;
+    private static final int DEFAULT_RADIUS = 10;
```

Comment:
The `MAX` and `MIN` constants are well-named, but it would be more descriptive to indicate what they are the maximum and minimum of, such as `MAX_SIDE_LENGTH` and `MIN_SIDE_LENGTH`.